MEETINGS.md

```diff
@@ -1,28 +1,32 @@
 # GitOps Working Group Meetings
-The GitOps Working group meets on the second Thursday of each month at 18:00 GMT - this meeting can be found in the [CNCF calendar](https://www.cncf.io/calendar/).
+The GitOps Working group typically meets on the second Thursday of each month at 18:00 GMT - this meeting can be found in the [CNCF calendar](https://www.cncf.io/calendar/).
 [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
 
 ## Upcoming Meetings
-### Thursday, March 11th at 18:00 GMT
-
+### Thursday, April 15th at 18:00 GMT
 - [Video Conference Link](https://zoom.us/my/cncfgitopswg?pwd=aE9FMjR0M1djd3huMlNUMWZXSXpyQT09)
 - Meeting Passcode: `77777`
 
 ## Past Meetings
+### Thursday, March 11th at 18:00 GMT
+
+- [Video Recording](https://zoom.us/rec/share/SYMUyXzmVF4eLmO9d2N3AOQ3ror5cJo361pnz_PtxUnDMio0vXRd-qpjqMHPnGeo.J_6sc5Mv8IIKi9jr?startTime=1615486031000)
+- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+
 ### Thursday, February 11th at 18:00 GMT
 
 - [Video Recording](https://weaveworks.zoom.us/rec/share/OPy5MSMzgIkPX1-PAdhwrnGGa82Lr5BuVpQ1iSK-sbHPph68ML1JNVQXRKlP09bw.bjqDnPKEA2W9D2HG)
 - Passcode: `fwPy8^fb`
-- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.w91ueqcmi7nz)
+- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
```

Comment:
@dj1k Just checking, you don't like the links that go directly to the heading for each week?

Reply:
Ah - I do, I was just removing the "edit" part as force of habit.